sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -631,6 +648,100 @@ protected void addFields() {
 			I18nProperties.getPrefixDescription(PersonDto.I18N_PREFIX, PersonDto.ADDITIONAL_DETAILS, "") + "\n"
 				+ I18nProperties.getDescription(Descriptions.descGdpr));
 		CssStyles.style(additionalDetails, CssStyles.CAPTION_HIDDEN);
+
+		hasGuardian = addField(PersonDto.HAS_GUARDIAN, CheckBox.class);
+		hasGuardian.addValueChangeListener(e -> updateHasGuardianCheckBox());
+
+		isIncapacitated = addField(PersonDto.IS_INCAPACITATED, CheckBox.class);
+		isIncapacitated.addValueChangeListener(e -> updateIsIncapacitatedCheckBox(true));
+
+		isEmancipated = addField(PersonDto.IS_EMANCIPATED, CheckBox.class);
+		isEmancipated.addValueChangeListener(e -> updateIsEmancipatedCheckBox());
+		hasGuardian.setEnabled(false);
+		minimumAdultAge = FacadeProvider.getConfigFacade().getMinimumAdultAge();//2 places
+		minimumEmancipatedAge = FacadeProvider.getConfigFacade().getMinimumEmancipatedAge();
+	}
+
+	private boolean isMinimumAgesInterval() {
+		int approximateAge = getApproximateAge();
+		if (approximateAge == -1){
+			return true;
+		}
+		return approximateAge > minimumEmancipatedAge && approximateAge < minimumAdultAge;
+	}
+
+	private int getApproximateAge() {
```

Comment:
please use the dedicated fields from PersonEditForm to retrieve the "aproximateAge": approximateAgeField, approximateAgeTypeSelect.